web/annotation_layer_builder.js

```diff
@@ -147,6 +174,29 @@ class AnnotationLayerBuilder {
     }
     this.div.hidden = true;
   }
+
+  #updatePresentationModeState(state) {
+    if (!this.div) {
+      return;
+    }
+    let disableFormElements = false;
+
+    switch (state) {
+      case PresentationModeState.FULLSCREEN:
+        disableFormElements = true;
+        break;
+      case PresentationModeState.NORMAL:
+        break;
+      default:
+        return;
+    }
+    for (const section of this.div.childNodes) {
```

Comment:
Isn't enough to add an `inert` in the `annotationLayer` div ?
I just tried:
```html
<div inert="true">
<input type="text" size="10"></input>
</div>
```
and the `input` is disabled.

Reply:
We still want *internal links* to be clickable, note the check just below, hence why this is done.